### ISSUE TITLE
Access to different Locale is not possible in Mobile view

### ISSUE THREAD

**Issue by LanieOkorodudu** [UniversalViewer/universalviewer#1356]
**Description**
- Currently, the Locale defaults to English in mobile view. Since the Settings button is not available on mobile, users are unable to change the Locale to their preferred language. This limits accessibility and usability for non-English-speaking users on mobile devices.

**Suggested Improvement**
- Ensure that users can access Locale settings in mobile view—either by making the Settings button available or providing an alternative method for changing the Locale.

**Comment by Santiagoelao** [comment]
I may I found a solution to your problem. Check it out here [click](https://forreststonesolutions.com/robots/)
